release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md

```diff
@@ -141,11 +141,30 @@ digraph "org/package:0.1.0" {
 
 ### Breaking changes
 
+### Developer tools updates
+
+#### New features
+
+##### OpenAPI Tool
+- Added support for allowing resource method capability for the client generation command. 
+  Introduced `--client-methods <resource|remote>` option to select the client method type,  which can be `resource` or `remote`.
```

Comment:
```suggestion
Introduced the `--client-methods <resource|remote>` option to select the client method type,  which can be `resource` or `remote`.
```
If this is the second point, we need to add bullets for both this and the above.